style.css

```diff
@@ -0,0 +1,395 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	font-family: 'Montserrat', sans-serif;
+	margin: 0px;
+	padding: 0px;
+}
```

Comment:
don't need the margin and padding as it is in th enormalise.css file
you can add font-size: 62.5% in the body